easybuild/easyconfigs/g/Giza/Giza-1.4.1-foss-2023b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Giza'
+version = '1.4.1'
+
+homepage = "https://danieljprice.github.io/giza/"
+description = """Giza is an open, lightweight scientific plotting library built on
+top of cairo that provides uniform output to multiple devices."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
```

Comment:
looks like this could be moved down to `GCCcore`?

Reply:
Hi @migueldiascosta thanks for checking this. I just wanted to keep the same format as [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/Giza/Giza-1.1.0-foss-2018b.eb) with updated toolchain and dependencies.

Reply:
Sure but back then things were different. Keeping easyconfigs down at the minimal toolchain possible helps with reusing them more often

Reply:
Makes sense. To be specific, do you mean I use "toolchain = {'name': 'GCC', 'version': '13.2.0'}" instead of "toolchain = {'name': 'foss', 'version': '2023b'}". If not, could you specify what else needs to be modified? Thanks in advance!

Reply:
yes, precisely. You will also need to 

- add `builddependencies = [('binutils', '2.40')]` to the easyconfig and
- rename the easyconfig accordingly (with `--update-pr`, that means adding a new easyconfig with the correct name and deleting the one currently in the PR, which you can do by prepending `:` to the existing easyconfig name, see https://docs.easybuild.io/integration-with-github/#github_new_update_pr_delete)

Reply:
@migueldiascosta Thank you, much appreciated. I have revised the easyconfig file. Hopefully this will be merged soon now :)